rpcs3/Emu/RSX/rsx_methods.cpp

```diff
@@ -59,19 +59,23 @@ namespace rsx
 			rsx->ctrl->ref.exchange(arg);
 		}
 
-		void semaphore_acquire(thread* rsx, u32 _reg, u32 arg)
+		void semaphore_acquire(thread* rsx, u32 addr, u32 arg)
```

Comment:
Why rename the input register to 'addr'? makes more sense to drop the arg entirely if its unused. I doubt it does anything measurable as an optimization and just harms readability here.